docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+services:
+  nginx:
+    image: "nginx:stable-alpine"
+    ports:
+      - 443:443
+    restart: unless-stopped
+    volumes:
+      - ./config/etc/nginx/nginx.conf:/etc/nginx/nginx.conf
+    depends_on: 
+      - registry
```

Comment:
This is backwards, the registry depends on nginx, no?

Also, we need to add something like https://github.com/Seneca-CDOT/telescope/blob/master/docker/production.yml#L139 to have nginx pick up the renewed certs when they get updated.

Reply:
I had added that on @manekenpix recommendation, I can switch it around though?

Reply:
Also, about updating the certs. Would I need to add a template file? and then just get rid of these env variables? 
![image](https://user-images.githubusercontent.com/59970326/156861995-14543a00-6add-4391-bc68-f2b6c4f48a9b.png)

Reply:
@humphd I think this is ok. if the registry is up but nginx is still booting, we could consider the state of the whole thing as down or booting, but not the other way around (we'd have nginx forwarding requests to nobody, which I think it's a weird state).

Reply:
Also, if we don't define an upstream in the nginx config file and nginx boots before the registry, nginx will probably crash (like in telescope).

Reply:
@TDDR Yes, you can drop the `envsubst` bit and just restart the nginx service.

@manekenpix OK, sounds good.